ceilometer/manifests/client/compute.pp

```diff
@@ -0,0 +1,42 @@
+#
+# Installs the ceilometer python library.
+#
+# == parameters
+#  [*ensure*]
+#    ensure state for pachage.
+#
+class ceilometer::client::compute (
+  $ensure = 'present'
+) {
+
+  include ::ceilometer::params
+
+  package { 'python-ceilometerclient':
```

Comment:
This package is also defined in ceilometer/manifests/client.pp 

Is there a good reason to create client/compute.pp instead of just modifying client.pp?   I don't see anything here that looks overly specific to compute hosts.